Makefile

```diff
@@ -2,13 +2,21 @@ start-deps:
 	docker-compose up -d
 	direnv reload
 
+start: start-deps
+	yarn build && node lib/entrypoint/graphql.js | yarn pino-pretty -c -l
+
+watch:
+	yarn nodemon -V -e ts,graphql -w ./src -x make start
+
 clean-deps:
 	docker-compose rm -sfv
 
 reset-deps: clean-deps start-deps
 
+restart-integration: reset-deps integration
```

Comment:
Is there a need for both `restart-integration` and `integration`? Serious question in your workflow do you ever need both? If not I think we can just use `integration`:
```
integration: restart-deps
  . ./.envrc && yarn jest --forceExit --bail --runInBand --verbose $$TEST | yarn pino-pretty -c -l
```

Reply:
Done